index.html

```diff
@@ -40,7 +40,7 @@
                 <a class="nav-link active" href="index.html">Home</a>
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="index.html">Chris' blog</a>
+                <a class="nav-link" href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" target="_blank">Chris' blog</a>
```

Comment:
fun choice for temp video link thanks!     since I don't have another place to put this request... when you have search results,  when you place the mouse cursor over the UL list elements, the cursor it the  text cursor.   You can use css hover style for those LI elements to show the "click" hand cursor instead.  My screenshot had trouble capturing the I text cursor.
![image](https://user-images.githubusercontent.com/651674/126731017-0ebe2119-0317-454d-94d1-fe9f176b8804.png)


So this is what I'm talking about doing:
https://www.tutorialrepublic.com/faq/how-to-change-the-cursor-into-a-hand-pointer-on-hover-using-css.php

![image](https://user-images.githubusercontent.com/651674/126730920-682007ea-d966-42fb-a2f7-4a9d1eaf5f14.png)